sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -76,14 +76,15 @@ case class CreateFunctionCommand(
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
     val func = CatalogFunction(FunctionIdentifier(functionName, databaseName), className, resources)
+    catalog.loadFunctionResources(resources)
+    // We fail fast if function class is not exists.
+    catalog.requireFunctionClassExists(func)
```

Comment:
I think we only need to do this for non-temp functions.

Reply:
Yeah, I also think about it.